trustedservices/rpc.go

```diff
@@ -5,19 +5,24 @@ package trustedservices
 
 import (
 	"context"
+	"errors"
 	"reflect"
 
 	"github.com/veraison/common"
 )
 
 func NewRPCServerParamStore() (*common.ParamStore, error) {
 	store := common.NewParamStore("vts-http")
+	if store == nil {
+		return nil, errors.New("param store initialization failed")
+	}
 	err := PopulateRPCServerParams(store)
 	return store, err
 }
 
 func PopulateRPCServerParams(store *common.ParamStore) error {
 	return store.AddParamDefinitions(map[string]*common.ParamDescription{
+		// NOTE(tho) should we use Path as map key?  (It'd make debug messages clearer)
```

Comment:
This would tie a param to a specific path within a config file. I explicitly tired to avoid that (hence referring to it by a name, rather than path).  A config file is tied to a particular deployment/configuration of Veraison components; e.g., I would expect a config file for a single-binary build of integrated Veraison services to look different from the config file for VTS node.

This would also make it easier to embed Veraison config as a sub-tree in a larger service config file, or even to re-use setting from an existing config file to initialize Veraison components.

Reply:
I understand the principle.  While fiddling with the config file, though, I was getting errors referring the symbolic name which is only available in the code and (we should assume) invisible to the end user.  And that usability-wise is not particularly friendly :-)

Reply:
True, we should probably modify the error message to include the Path as well.